modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -31,7 +31,10 @@ namespace core {
     using GMat2 = std::tuple<GMat,GMat>;
     using GMat3 = std::tuple<GMat,GMat,GMat>; // FIXME: how to avoid this?
     using GMat4 = std::tuple<GMat,GMat,GMat,GMat>;
-    using GMatScalar = std::tuple<GMat, GScalar>;
+    using GMatScalar  = std::tuple<GMat, GScalar>;
+    using GRects      = cv::GArray<cv::Rect>;
+    using GDetections = std::tuple<GArray<Rect>,GArray<int>>;
+    using YoloAnchors = std::vector<float>;
```

Comment:
I thought you were planning to add a separate header for it, isn't it?

`core` is more or less a mirror of OpenCV `core` where there's no such operations.

Reply:
Parsers(SSD/Yolo) are in nnparsers.hpp